docs/overview.md

```diff
@@ -26,27 +24,27 @@ The skeleton of a model might look like this:
 import mesa
 
 class MyAgent(mesa.Agent):
-    def __init__(self, name, model):
-        super().__init__(name, model)
-        self.name = name
+    def __init__(self, model, age):
+        super().__init__(model)
+        self.age = age
 
     def step(self):
-        print("{} activated".format(self.name))
+        self.age += 1
+        print(f"Agent {self.unique_id} now is {self.age} years old")
         # Whatever else the agent does when activated
 
 class MyModel(mesa.Model):
     def __init__(self, n_agents):
         super().__init__()
-        self.schedule = mesa.time.RandomActivation(self)
         self.grid = mesa.space.MultiGrid(10, 10, torus=True)
```

Comment:
given the desire to switch to new experimental spaces, do we want to use an old space here?

Reply:
Let's discuss integration of the cell spaces in the next dev call.